src/Type/Php/DateIntervalConstructorThrowTypeExtension.php

```diff
@@ -24,18 +26,21 @@ public function getThrowTypeFromStaticMethodCall(MethodReflection $methodReflect
 			return $methodReflection->getThrowType();
 		}
 
-		$arg = $methodCall->args[0]->value;
-		$constantStrings = TypeUtils::getConstantStrings($scope->getType($arg));
```

Comment:
If the arg had valid constant strings and another type, the extension was considering that now exception was thrown since only the constant strings were checked, isn't it ?